index.html

```diff
@@ -220,18 +220,30 @@ <h2>
               </li>
               <li>
                 <p>
-                  If controller is not <code>null</code>, run the following steps:
+                  If <var>controller</var> is not <code>null</code>, run the following steps:
                 </p>
-                  <ol>
-                    <li>
-                      If <var>controller</var>.{{CaptureController/[[isBound]]}}
+                <ol>
+                  <li>
+                    <p>
+                      If <var>controller</var>.{{CaptureController/[[IsBound]]}}
                       is <code>true</code>, return a promise [=reject|rejected=] with
                       a {{DOMException}} object whose {{DOMException/name}} attribute has
                       the value {{InvalidStateError}}.
-                    </li>
-                    <li>
-                      Set <var>controller</var>.{{CaptureController/[[isBound]]}} to <code>true</code>.
-                    </li>
+                    </p>
+                  </li>
+                  <li>
+                    <p>
+                      Set <var>controller</var>.{{CaptureController/[[IsBound]]}} to <code>true</code>.
+                    </p>
+                  </li>
+                  <li>
+                    <p>
+                      If the <a>current settings object</a>'s [=relevant global object=]'s
+                      [=associated `Document`=] ever <a data-cite="!HTML/#focused">loses focus</a>,
```

Comment:
Shall we consider [page visibility](https://html.spec.whatwg.org/multipage/interaction.html#page-visibility) instead of document focus?

Reply:
getDisplayMedia uses this condition.
> If the [current settings object](https://html.spec.whatwg.org/multipage/webappapis.html#current-settings-object)'s [relevant global object](https://html.spec.whatwg.org/multipage/webappapis.html#concept-relevant-global)'s [associated Document](https://html.spec.whatwg.org/multipage/window-object.html#concept-document-window) is NOT [fully active](https://html.spec.whatwg.org/multipage/browsers.html#fully-active) or does NOT [have focus](https://html.spec.whatwg.org/multipage/#gains-focus), return a promise [rejected](https://webidl.spec.whatwg.org/#reject) with a [DOMException](https://webidl.spec.whatwg.org/#idl-DOMException) object whose [name](https://webidl.spec.whatwg.org/#dom-domexception-name) attribute has the value [InvalidStateError](https://webidl.spec.whatwg.org/#invalidstateerror).

Reply:
Let me know if you have a fix in mind that's uncontroversial enough to make it into this PR. Otherwise, I expect we'll be fixing as a follow-up. (I'm resolving this conversation for now, but that's a reversible action.)